### PR TITLE
[EKG-99345] Fix Exception when getting a schema for offline table

### PR DESCRIPTION
**Pull request by mcvsubbu**

### CODE REVIEW

File: pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotTableSchema.java [hunk 1]
```diff
@@ -75,7 +73,10 @@ private Representation getTableSchema(
       try {
         config = _pinotHelixResourceManager.getTableConfig(tableName, TableType.OFFLINE);
         String schemaName = config.getValidationConfig().getSchemaName();
-        Schema schema = _pinotHelixResourceManager.getSchema(schemaName);
+        Schema schema = null;
+        if (schemaName != null && schemaName.length() > 0) {
```
Comment (mayankshriv) [inline pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotTableSchema.java]: May be use isEmpty() over length() > 0?

**Comment by mcvsubbu** [comment]
Done

File: pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotTableSchema.java [hunk 2]
```diff
@@ -75,7 +73,10 @@ private Representation getTableSchema(
       try {
         config = _pinotHelixResourceManager.getTableConfig(tableName, TableType.OFFLINE);
         String schemaName = config.getValidationConfig().getSchemaName();
-        Schema schema = _pinotHelixResourceManager.getSchema(schemaName);
+        Schema schema = null;
+        if (schemaName != null && schemaName.isEmpty()) {
```
Comment (jfim) [inline pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotTableSchema.java]: I assume that should be `!schemaName.isEmpty()` :)

**Comment by mcvsubbu** [comment]
Yup, right you are.

-Subbu

On Sat, Nov 12, 2016 at 12:26 AM, Jean-François Im <notifications@github.com

**Pull request merged**